packages/gatsby-plugin-google-gtag/src/gatsby-ssr.js

```diff
@@ -0,0 +1,73 @@
+import React from 'react'
+
+export const onRenderBody = (
+  { setHeadComponents, setPostBodyComponents },
+  pluginOptions
+) => {
+  if (process.env.NODE_ENV !== `production`) return null
+
+  const firstTrackingId =
+    pluginOptions.trackingIds && pluginOptions.trackingIds.length
+      ? pluginOptions.trackingIds[0]
+      : ``
+
+  const excludeGtagPaths = []
+  if (typeof pluginOptions.pluginConfig.exclude !== `undefined`) {
+    const Minimatch = require(`minimatch`).Minimatch
```

Comment:
Maybe hoist this to the top for clarity? (even if we lose the benefit of requiring only when it's needed)

```js
import { Minimatch } from 'minimatch'
```